lib/App/DuckPAN/Fathead.pm

```diff
@@ -29,6 +29,39 @@ sub _trigger_selected {
 	return $dir;
 }
 
+has _trigger_words => (
+	is      => 'ro',
+	builder => 1,
+	lazy    => 1,
+);
+
+sub _build__trigger_words {
+	my ($self) = @_;
+	my $tf = 'trigger_words.txt';
+	return [] unless $self->has_selected;
+	my $file = path("lib/fathead/", $self->selected, $tf);
+	unless ($file->exists){
+		my $full_path = $file->realpath;
+		$self->app->emit_debug("No $tf was found in $full_path");
+		return [];
+	}
+	chomp (my @words = $file->lines);
+	return \@words;
+}
+
+has _trigger_re => (
+	is      => 'ro',
+	lazy    => 1,
+	builder => 1,
+);
+
+sub _build__trigger_re {
+	my ($self) = @_;
+	my @words = @{$self->_trigger_words};
+	my $text = '\b(?:' . (join '|', map { quotemeta $_ } @words) . ')\b';
+	return qr/$text/i;
```

Comment:
nit: I think it would be clearer to put the `\b` and `(?:` into the `qr/`

```perl
my $text = join '|', map { quotemeta $_ } @words;
return qr/\b(?:$text)\b/;
```